### DISCUSSION TITLE
SidebarText op has old style string ID

### DISCUSSION THREAD

**Discussion by mariomatasystems** [cables-gl/cables_docs#910] [Feature Requests] [answered]
<img width="494" alt="Screenshot 2024-11-13 at 11 03 22" src="https://github.com/user-attachments/assets/32acd0ec-b6b3-42cf-99df-a07f5fa75aa2">

ID should be a red string input

**Comment by pandrr** [comment, marked as answer]
done on dev.cables.gl, please make issues next time, not discussions for such clear bugs
